translations/client_pl.ts

```diff
@@ -3740,7 +3740,7 @@ To nowy, eksperymentalny tryb. Jeśli zdecydujesz się z niego skorzystać, zgł
     <message>
         <location filename="../src/gui/socketapi.cpp" line="783"/>
         <source>Copy private link to clipboard</source>
-        <translation>Kopiuj link prywatny do schowka</translation>
+        <translation>Skopiuj link prywatny do schowka</translation>
```

Comment:
@elsiehupp I am not sure of the automated way to undo this, but, you can undo it manually by taking this file translations/client_pl.ts from master and replace your local file with it. It will then be removed from the changed files in this PR.

With that said, you can also try doing:

git checkout -- translations/client_pl.ts 

to get the git's master version of this file.
I never used this, so, not sure if this is going to work. You can always undo changes manually if it fails.

Reply:
I was able to do it with the following:
```
% git pull
% git reset --soft upstream/master
[Discard extra files and commit in VS Code]
% git push origin +maximize-settings-window
```